scriptmodules/emulators/dosbox-staging.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="dosbox-staging"
+rp_module_desc="DOS emulator (enhanced and modern DOSBox fork)"
+rp_module_help="ROM Extensions: .bat .com .exe .sh .conf\n\nCopy your DOS games to $romdir/pc"
+rp_module_licence="GPL2 https://github.com/dosbox-staging/dosbox-staging/blob/master/COPYING"
+rp_module_repo="git https://github.com/dosbox-staging/dosbox-staging.git master"
+rp_module_section="exp"
+
+function depends_dosbox-staging() {
+    getDepends ccache build-essential meson ninja-build libasound2-dev libpng-dev libsdl2-dev libsdl2-net-dev libopusfile-dev libfluidsynth-dev
```

Comment:
`ccache` and `libfluidsynth-dev` shouldn't be listed. 

Since `libfluidsynth` is statically linked, I guess we should list the dependencies for `libfluidsynth`. `libglib2.0-dev` seems to be needed, other libraries may be optional since the `fluidsynth` build doesn't need Jack/ALSA/PulseAudio/etc. support.

`cmake` is also needed by the `fluidsynth` build. While technically it's installed when adding `emulationstation`, it's better to specifically add it.

Reply:
`ccache` and `libfluidsynth-dev` both removed from list. I've added `cmake` and `libglib2.0-dev` instead + sorted list. For further reference Fluidsynth requirements are being listed [here](https://github.com/FluidSynth/fluidsynth/wiki/BuildingWithCMake#requirements).